pkgs/servers/sql/cockroachdb/default.nix

```diff
@@ -1,70 +1,52 @@
-{ lib, stdenv, buildGoPackage, fetchurl
-, cmake, xz, which, autoconf
-, ncurses6, libedit, libunwind
-, installShellFiles
-, removeReferencesTo, go
+{ lib
+, stdenv
+, buildFHSEnv
 }:
 
 let
-  darwinDeps = [ libunwind libedit ];
-  linuxDeps  = [ ncurses6 ];
-
-  buildInputs = if stdenv.isDarwin then darwinDeps else linuxDeps;
-  nativeBuildInputs = [ installShellFiles cmake xz which autoconf ];
-
-in
-buildGoPackage rec {
-  pname = "cockroach";
+  name = "cockroach";
   version = "20.1.8";
```

Comment:
Isn't that a very old version? Or should we bump versions in a later PR?

Reply:
I would suggest du bump it later.